packages/hdwallet-ledger/package.json

```diff
@@ -19,14 +19,15 @@
     "@ethereumjs/tx": "^3.3.0",
     "@ledgerhq/hw-app-cosmos": "^6.29.1",
     "@shapeshiftoss/bitcoinjs-lib": "7.0.0-shapeshift.0",
-    "@shapeshiftoss/hdwallet-core": "1.59.2",
+    "@shapeshiftoss/hdwallet-core": "1.59.3",
     "@solana/web3.js": "1.95.8",
     "base64-js": "^1.5.1",
     "bchaddrjs": "^0.4.4",
     "bitcoinjs-message": "^2.0.0",
     "bs58check": "2.1.2",
     "ethereumjs-tx": "1.3.7",
     "ethereumjs-util": "^6.1.0",
+    "ethers": "5.7.2",
```

Comment:
🧠

Reply:
Yeah I was about to say that this is the actual diff